spec/models/manageiq/providers/oracle_cloud/cloud_manager/refresher_spec.rb

```diff
@@ -184,7 +41,7 @@ def assert_specific_cloud_volume
         with_vcr("vm_target") { refresh(ems_event.manager_refresh_targets) }
 
         vm.reload
-        expect(vm.raw_power_state).to eq("STOPPED")
+        assert_specific_instance(:expected_raw_power_state => "STOPPED", :expected_power_state => "off")
```

Comment:
The existing targeted refresh spec didn't show the failures of having the wrong strategy, and this is a better test anyway